.cirrus.yml

```diff
@@ -108,6 +108,7 @@ gating_task:
     gate_script:
         - '/usr/local/bin/entrypoint.sh validate'
         - '/usr/local/bin/entrypoint.sh lint'
+        - '/usr/local/bin/entrypoint.sh clean podman BUILDTAGS="exclude_graphdriver_devicemapper selinux seccomp"'
```

Comment:
best make this it's own script section.  It'll be easier to debug in the WebUI what broke.  i.e. The validate + Lint checks have lots of output.  So for example:

Reply:
(meaning, the varlink check you want to add.)

Reply:
Like this?

Reply:
Naa, I mean like:

```yaml
    varlink_script:
        - '/usr/local/bin/entrypoint.sh clean podman BUILDTAGS="exclude_graphdriver_devicemapper selinux seccomp"'
```

That way it ends up inside it's own dedicated drop-down thing in the WebUI.  That will make it quick to see at a glance what broke (lint vs something else).

Reply:
(or I'm misunderstanding something)

Reply:
OK I think I actually got it this time

Reply:
yep, u got it :smile: